app.py

```diff
@@ -203,6 +206,10 @@ def update_graph_content(selected_model):
             dmc.Center(dmc.Text("PPD = 3")),
             dmc.Center(dmc.Text("SET = 29.9")),
         ]
+    else:
```

Comment:
This is not needed since the user cannot select something that is not in the dropdown